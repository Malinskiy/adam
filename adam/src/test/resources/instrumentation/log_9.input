INSTRUMENTATION_STATUS: class=com.bofa.android.adapoc.home.HomeActivityTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stream=
com.bofa.android.adapoc.home.HomeActivityTest:
INSTRUMENTATION_STATUS: test=verify01_homeActivity_displayed
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: description=TestDescription
INSTRUMENTATION_STATUS: displayName=TestDisplayName
INSTRUMENTATION_STATUS: dsaProfileName=FooBar
INSTRUMENTATION_STATUS: module=FooBar
INSTRUMENTATION_STATUS: storyIdentifier=TestIdentifier
INSTRUMENTATION_STATUS: testIdentifier=Ex-1023
INSTRUMENTATION_STATUS_CODE: 2
INSTRUMENTATION_STATUS: class=com.bofa.android.adapoc.home.HomeActivityTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stream=.
INSTRUMENTATION_STATUS: test=verify01_homeActivity_displayed
INSTRUMENTATION_STATUS_CODE: 0
INSTRUMENTATION_RESULT: stream=

Time: 17.084

OK (1 test)


INSTRUMENTATION_CODE: -1
