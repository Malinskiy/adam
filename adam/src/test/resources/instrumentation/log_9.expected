TestRunStartedEvent(testCount=1)
TestStarted(id=TestIdentifier(className=com.bofa.android.adapoc.home.HomeActivityTest, testName=verify01_homeActivity_displayed))
TestEnded(id=TestIdentifier(className=com.bofa.android.adapoc.home.HomeActivityTest, testName=verify01_homeActivity_displayed), metrics={description=TestDescription, displayName=TestDisplayName, dsaProfileName=FooBar, module=FooBar, storyIdentifier=TestIdentifier, testIdentifier=Ex-1023})
TestRunEnded(elapsedTimeMillis=17084, metrics={})
